applications/StructuralMechanicsApplication/python_scripts/adjoint_structural_solver.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
can you please use the same header as for the other python files in StructuralMechanics?